src/Http/CookieFactory.php

```diff
@@ -79,7 +80,7 @@ public function make($name, $value = null, $maxAge = null)
         if ($maxAge) {
             $cookie = $cookie
                 ->withMaxAge($maxAge)
-                ->withExpires(time() + $maxAge);
+                ->withExpires(Carbon::now()->timestamp + $maxAge);
```

Comment:
Why the use of Carbon instead of `time()`?

Reply:
Reverted in https://github.com/flarum/core/pull/1344/commits/b2e873ba7bc0c31b1d7fc1c0e96d347c93bebc5f